ws-nextjs-app/pages/[service]/send/[id]/FormField/styles.ts

```diff
@@ -24,6 +19,9 @@ export default {
           marginBottom: `${spacings.TRIPLE}rem`,
         },
       },
+      '&:nth-of-type(5) label': {
+        marginBottom: '0',
+      },
```

Comment:
Since the form fields are dynamic, this will only apply this styling to the 5th label. What styling do we expect when there are 4 or less or more than 5 form fields?

Reply:
ahh i see, sorry had assumed it would always be 5, think this can be changed to 'last-of-type'  or something similar instead ..